docs/glossary.md

```diff
@@ -1,11 +1,13 @@
 ## Wikijump Glossary
 
 There are a number of concepts within Wikidot, which can take on a number of different names in casual usage.
-To standardize their usage during development and within the API, this document will describe the standard name, Wikidot's name (if different), and any other aliases.
+To standardize their usage during development and within the API, this document will describe the standard API name, Wikidot's name (if different), and any other aliases.
+
+Please use the Wikijump API term when writing technical documentation, writing code, or utilizing the API. The aim is clarity and disambiguity.
 
 Definitions are not intended to be exhaustive, but rather quickly summarize what it is for recognition. If further details are needed, add a link to a more detailed resource.
 
-| Official Term  | Wikidot Term | Other Terms | Definition |
+| API Term       | Wikidot Term | Other Terms | Definition |
```

Comment:
```suggestion
| Wikijump Term  | Wikidot Term | Other Terms | Definition |
```
We're going to be using these terms in contexts outside the API, so being as generic as possible is in our best interests here. Also calling it "Wikijump Term" is a direct, clear parallel to "Wikidot Term" which should hopefully avoid any confusion at all